src/authextension.ts

```diff
@@ -9,13 +9,13 @@ import { Utils } from './Utils';
 export module authextension {
   export let setContextData: any;
 
-  setContextData = (context_f8_access_routes, context_f8_3scale_user_key) => {
-    Apiendpoint.STACK_API_URL = Apiendpoint.STAGE_API_URL + '/api/v2/';
-    Apiendpoint.STACK_API_USER_KEY = Apiendpoint.STAGE_THREE_SCALE_USER_TOKEN;
-    Apiendpoint.OSIO_ROUTE_URL = Apiendpoint.STAGE_API_URL;
+  setContextData = (apiRoutes) => {
+    Apiendpoint.STACK_API_URL = apiRoutes.host + '/api/v2/';
+    Apiendpoint.STACK_API_USER_KEY = apiRoutes.apiKey;
+    Apiendpoint.OSIO_ROUTE_URL = apiRoutes.host;
```

Comment:
Are we still using the OSIO route?

Reply:
OSIO_ROUTE_URL points to same 3scale app host. +1 to remove OSIO_ROUTE_URL.